duneana/AnaTree/AnalysisTree_module.cc

```diff
@@ -4233,12 +4246,12 @@ void dune::AnalysisTree::analyze(const art::Event& evt)
     // Get a PFParticle-to-track map.
     lar_pandora::TrackVector allPfParticleTracks;
     lar_pandora::PFParticlesToTracks pfParticleToTrackMap;
-    lar_pandora::LArPandoraHelper::CollectTracks(evt, fPFParticleModuleLabel, allPfParticleTracks, pfParticleToTrackMap);
+    lar_pandora::LArPandoraHelper::CollectTracks(evt, fTrackModuleLabel[0], allPfParticleTracks, pfParticleToTrackMap);
```

Comment:
If I understood correctly, there is only one PFP branch that can only be associated to one label.
However, tracks and showers can have more then one (for example, in the fcl file: `[ "pmtrack", "pmtrackdc", "pandoraTrack", "pandoraTrackdc" ]`). Then I took the first label only. This works in the case that someone is using only `pandoraTrack` and `pandoraShower` as label. 

The only option  I see to have a PFP association with all the possible labels would be to create different branches of PFP for each label. But this would be too complicated (for me) and would change the anatree too much.

Reply:
Exactly, 

In my case I am only using `["pandoraTrack"]`, so it was fine

Reply:
I think this is a tricky one to generally solve.  Given that pmtrack etc are not maintained anymore then the most comfortable solution is to do what you've done, but also change the standard order in the fcl to have `pandoraTrack` first.  Could you add that in?

Reply:
Done !

I also added the labels for saving the PFP information as it was missing in the standard fcl file.